api/src/index.ts

```diff
@@ -65,26 +61,44 @@ async function run() {
     config.minterAddress
   );
 
-  const blockCursorService = new BlockCursorService();
-
-  const saleOfferWorker = new SaleOfferHandler(
-    marketService,
-    blockCursorService,
-    flowService
-  );
-
-  const app = initApp(
-    knexInstance,
-    kibblesService,
-    kittyItemsService,
-    marketService
-  );
-
-  app.listen(config.port, () => {
-    console.log(`Listening on port ${config.port}!`);
-  });
+  // Make sure we're pointing to the correct Flow Access API.
+  fcl.config().put("accessNode.api", config.accessApi);
 
-  saleOfferWorker.run();
+  if (argv.worker) {
+    Model.knex(knexInstance);
```

Comment:
Ah, I guess this had to be added here because otherwise it's only called in `initApp`, which doesn't get invoked when running the worker, right?

I think it'd be worth it to create a separate `api/src/db.js` file that handles the Knex + objection ORM setup -- that way both the API and worker can depend on the same shared setup code.

Reply:
Agreed. Thisfeel out of place. Was my 'get it working`' code. I think we could have a simple `initWorker` function.

Reply:
Still have to get to this.